brownie/utils/hex.py

```diff
@@ -0,0 +1,29 @@
+from importlib.metadata import version
+from typing import Final
+
+from eth_typing import HexStr
+from hexbytes import HexBytes
+
+HEXBYTES_LT_1_0_0: Final = tuple(int(i) for i in version("hexbytes").split(".")) < (
+    1,
+    0,
+    0,
+)
+"""
+A boolean constant that indicates if the version of your `hexbytes` package is less than 1.0.0.
+
+This exists to address a breaking change in hexbytes v1 and allow brownie to be used with
+either version.
+"""
+
+if HEXBYTES_LT_1_0_0:
+
+    def hexstring(value: bytes) -> HexStr:
+        """Convert a bytes value to a hexstring on hexbytes<1."""
+        return HexBytes(value).hex()
+
+else:
+
+    def hexstring(value: bytes) -> HexStr:
+        """Convert a bytes value to a hexstring on hexbytes>=1."""
+        return f"0x{HexBytes(value).hex()}"
```

Comment:
same as before (and a lot of other places we could surely apply this, unless you have a specific reason for doing it the way you are)

```python
def hexstring(value: bytes) -> HexStr:
    return f"0x{HexBytes(value).hex().removeprefix('0x')}"
```

think we could get rid of this ugly `HEXBYTES_LT_1_0_0` altogether

Reply:
was just for speed since my server is running thru these functions 24/7

we can probably get rid of the constant in the other files while keeping the optimized def in this file, wdyt?

Reply:
wait I think if I just take out the call to HexBytes here, and just build the hexstring from bytes.hex, we can use this util in all the places we currently have HEXBYTES_LT_1_0_0

That's my new plan, I'll implement if you agree